.github/dependabot.yaml

```diff
@@ -1,9 +1,58 @@
 version: 2
 updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+      day: "friday"
+      time: "08:00"
+      timezone: "Europe/London"
+    labels:
+      - "dependencies"
+      - "github-actions"
+
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+      day: "friday"
+      time: "08:00"
+      timezone: "Europe/London"
+    versioning-strategy: increase-if-necessary
+    labels:
+      - "dependencies"
+      - "node"
+    groups:
+      development-dependencies:
+        dependency-type: "development"
+        patterns:
+          - "@wdio*"
+          - "@babel*"
+          - "eslint*"
+          - "json*"
+          - "jsrsasign*"
+          - "lint-staged*"
+          - "livereload*"
+          - "node-forge*"
+          - "prettier*"
+          - "typescript*"
+          - "uuid*"
+          - "webdriverio*"
+
   - package-ecosystem: "pip"
     directory: "/"
     schedule:
-      interval: "daily"
+      interval: "weekly"
+      day: "friday"
+      time: "08:00"
+      timezone: "Europe/London"
+    target-branch: main
```

Comment:
Is it worth popping in a comment for this line, as it's the workaround for having 2 pip ecosystems?